dolittle/Studio
# Bringing view online

# Todo
- [x] Make it clearer that you cant do something.

# How to test
- Start with a clean cluster (or without raw-data-log and purchase-order)
- Click Microservices
- Click Create new microservice
- Click Create within the Purchase order api card
- Select Infor graphic
- Enter name
- Click next
- Enter username
- Enter password
- Click Next
- At this point, we send the request to the backend and redirect the frontend.
- Observe platform-api crashed (we have a bug for that)
- Restart platform-api
- Refresh browser
- Observe Screen looks similar
- Click back button to try and goto "Provide Name"
- Observe not able to
- Click on "Health Status" to confirm it works.


## Summary

- Bringing the view screen online
- Displays message when save is triggered, does not save for now, as we have not handled the update

# Reference
- https://app.asana.com/0/1200899594200551/1201018124798231/f
